app/src/main/AndroidManifest.xml

```diff
@@ -31,6 +31,10 @@
             <meta-data android:name="android.support.PARENT_ACTIVITY"
                 android:value=".Activities.MainActivity" />
         </activity>
+        <meta-data
+            android:name="io.fabric.ApiKey"
+            android:value="45894d9e8a6bc3b8513651d6de36159e2c836e51"
+            />
     </application>
-
+    <uses-permission android:name="android.permission.INTERNET" />
```

Comment:
Move this above `<application>`. I don't think it should matter(?) but just so we are consistent with the Manifest file structure as seen here: https://developer.android.com/guide/topics/manifest/manifest-intro.html#filestruct